packages/grafana-toolkit/src/cli/tasks/toolkit.build.ts

```diff
@@ -113,9 +85,7 @@ const toolkitBuildTaskRunner: TaskRunner<ToolkitBuildOptions> = async () => {
   const pkg = require(`${cwd}/package.json`);
   console.log(chalk.yellow(`Building ${pkg.name} (package.json version: ${pkg.version})`));
 
-  await clean();
```

Comment:
**Question:** probably I am wrong, but isn't this something that was/is used by plugins? 🤔

Reply:
Will double check but I hope it isn't used by plugins! The task associated to these functions specifically builds toolkit from what I could tell. 🤔

Reply:
I double checked here and this code is only for building toolkit itself.

```js
import { program } from 'commander';
import { toolkitBuildTask } from './tasks/toolkit.build';

// ...
program
  .command("toolkit:build")
  .description("Prepares grafana/toolkit dist package")
  .action(async (cmd) => {
    await execTask(toolkitBuildTask)({});
  });

```

However while doing some more testing I noticed removing `clean` from this task was a bad idea. Have put it back to prevent multiple build failures occurring if `dist` already exists.